src/api/python/cvc5.pxi

```diff
@@ -1478,16 +1504,39 @@ cdef class Solver:
             raise ValueError("Unexpected inputs to mkBitVector")
         return term
 
-    def mkFiniteFieldElem(self, value, Sort sort):
+    def mkFiniteFieldElem(self, value, Sort sort, *args):
         """
             Create finite field value.
 
+            Supports the following arguments:
+            
+            - ``Term mkFiniteFieldElem(value, Sort sort)``
+            - ``Term mkFiniteFieldElem(string value, Sort sort, int base)``
+
             :return: A Term representing a finite field value.
-            :param value: The value of the element's integer representation.
+            :param value: The value of the element's integer or string in base 10 representation.
```

Comment:
Is base 10 really a requirement here? Isn't parameter `base` what determines the base the string is given in?

Reply:
The default method that is called with one parameter is `Sort mkFiniteFieldElem(const string& s, Sort)` which works only with a base 10 representation. I used mkBitVector creation methods as a reference to this.